src/coord/src/catalog/storage.rs

```diff
@@ -518,6 +524,7 @@ impl<S: Append> Connection<S> {
             introspection_sources: TableTransaction::new(introspection_sources, None, |_a, _b| {
                 false
             }),
+            system_gid_mappings: TableTransaction::new(system_gid_mappings, None, |_a, _b| false),
```

Comment:
You need to add this to `commit()` so the updates get persisted.

Reply:
Thanks, just updated